hikari/api/rest.py

```diff
@@ -291,21 +294,25 @@ async def edit_channel(
 
             This only applies to forum channels.
         archived : hikari.undefined.UndefinedOr[bool]
-            If provided, whether to archive or unarchive this thread channel.
+            If provided, the new archived state for the thread. This only
+            applies to threads.
         locked : hikari.undefined.UndefinedOr[bool]
-            If provided, whether to lock or unlock this thread channel.
+            If provided, the new locked state for the thread. This only applies
+            to threads.
 
             If it's locked then only people with `MANAGE_THREADS` can unarchive it.
         invitable : undefined.UndefinedOr[bool]
-            If provided, whether non-moderators should be able to add other non-moderators to the thread.
-
-            This only applies to private threads.
+            If provided, the new setting for whether non-moderators can invite
+            new members to a private thread. This only applies to threads.
```

Comment:
Why was the wording changed for this one?

Reply:
Make the doc consistent with the docs for `GuildChannel.edit` which is taken directly from the Discord Resources docs.

Reply:
I think i like the other wording better, but would appreciate other comments :)